src/app/ddDrakeModelOH.cpp

```diff
@@ -445,12 +445,12 @@ class URDFRigidBodyTreeVTK : public RigidBodyTree
 
     RigidBodyTree* model = this;
 
-    const std::shared_ptr<RigidBody> worldBody = model->bodies[0];
```

Comment:
I don't think the OH model should be touched at all - it uses an older, fixed version of Drake and thus should not be updated if the upstream API changes. Only the ddDrakeModel should be updated.

Reply:
Done. Sorry about that!